From 4eaafe67de79fb63ebf61f031a97bbc0ed6a8891 Mon Sep 17 00:00:00 2001
From: Albert Astals Cid <aacid@kde.org>
Date: Wed, 29 Feb 2012 23:22:34 +0100
Subject: [PATCH] Change nnnnnn to number

This way people won't expect it to be six fixed digits
Bug #46708
---
 utils/pdftoppm.1 |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/pdftoppm.1 b/utils/pdftoppm.1
index c7d5d6c..2321d6d 100644
--- a/utils/pdftoppm.1
+++ b/utils/pdftoppm.1
@@ -17,9 +17,9 @@ Graymap (PGM) format, or monochrome image files in Portable Bitmap
 Pdftoppm reads the PDF file,
 .IR PDF-file ,
 and writes one PPM file for each page,
-.IR PPM-root - nnnnnn .ppm,
+.IR PPM-root - number .ppm,
 where
-.I nnnnnn
+.I number
 is the page number.
 .SH OPTIONS
 .TP
-- 
1.7.10

